jupyter_book/commands/__init__.py

```diff
@@ -242,6 +242,8 @@ def build(
         # Builder-specific overrides
         if builder == "pdfhtml":
             config_overrides["html_theme_options"] = {"single_page": True}
+        if builder == "pdflatex":
```

Comment:
Is this just temporarily because of the sphinxcontrib-bibtex bug? or not?

Reply:
Thanks @choldgraf for pointing this out. I will remove this one, as we have already merged a temporary fix for sphinxcontrib-bibtex